scripts/NuGetUtilities.psm1

```diff
@@ -22,14 +22,22 @@ $PackageSource = "https://nuget.org"
 $PackageOutputDirectoryRoot = Join-Path $BinRoot NuGet
 
 function Get-PackageVersion([switch]$previous) {
-    $versionPrefix, $versionSuffix = & $PSScriptRoot\Get-VersionConstants.ps1 -Previous:$previous
+    $versionPrefix, $versionSuffix, $packageVersionSuffix = & $PSScriptRoot\Get-VersionConstants.ps1 -Previous:$previous
     $version = $versionPrefix
-    if ($versionSuffix)
+    
+	if ($versionSuffix)
     {
         $version += "-$versionSuffix"
     }
 
-    $version
+    if ($packageVersionSuffix)
+    {
+        $version += $packageVersionSuffix
+    }
+
+   Write-Information "  *** RETURNING VERSION: $version."
```

Comment:
>Write [](start = 3, length = 5)

Remove.

Reply:
Oops.

---
In reply to: [259152044](https://github.com/Microsoft/sarif-sdk/pull/1304#discussion_r259152044) [](ancestors = 259152044)